include/opcode/riscv-opc.h

```diff
@@ -1557,8 +1557,21 @@ xxx - 00 x 10000 xxxxx --- xxxxx 0000111 VLSEG<nf>E<EEW>FF
 
 #define MATCH_VL1RV  0x02800007
 #define MASK_VL1RV   0xfff0707f
+#define MATCH_VL2RV  0x12800007
```

Comment:
Is this right?  The <nf> field is 3 bits, so it looks like it should be 0x2....  And similarly below the 0x3 should be 0x6 and the 0x7 should be 0xe.

Reply:
No it's wrong....  I’m sorry to have made such a stupid mistake....  Already fixed now.  And I also notice that we have a new mew encoding for vector load/store, these will be added by other patches.  Thanks.